api/mock/facade.go

```diff
@@ -84,6 +104,21 @@ func (f *Facade) GenerateTransaction(sender string, receiver string, value *big.
 	return f.GenerateTransactionHandler(sender, receiver, value, code)
 }
 
+// CreateTransaction is  mock implementation of a handler's CreateTransaction method
+func (f *Facade) CreateTransaction(
+	nonce uint64,
+	value *big.Int,
+	receiverHex string,
+	senderHex string,
+	gasPrice uint64,
+	gasLimit uint64,
+	data string,
+	signatureHex string,
+	challenge string,
+) (*transaction.Transaction, error) {
+	return f.CreateTransactionHandler(nonce, value, receiverHex, senderHex, gasPrice, gasLimit, data, signatureHex, challenge)
```

Comment:
After a func declaration like this, I would let an empty line after {

Reply:
done